browser/resources/settings/social_blocking_page/social_blocking_page.js

```diff
@@ -7,34 +7,5 @@
  * social blocking options
  */
 Polymer({
-  is: 'settings-social-blocking-page',
-
-  properties: {},
-
-  /** @private {?settings.SocialBlockingPageProxy} */
-  browserProxy_: null,
-
-  /** @override */
-  created: function() {
-    this.browserProxy_ = settings.SocialBlockingPageProxyImpl.getInstance();
-  },
-
-  /** @override */
-  ready: function() {
-    this.shouldShowRestart_ = this.shouldShowRestart_.bind(this)
-    this.restartBrowser_ = this.restartBrowser_.bind(this)
-  },
-
-  shouldShowRestart_: function(value) {
```

Comment:
@jumde this function and `setSignInEnabledAtNextStartup_` below have been removed from this .js but they are still referenced in the HTML. Were they meant to stay and just have their functionality moved here from the proxy?

Reply:
checking